test/integration/sanAgeDestroyedTest.js

```diff
@@ -0,0 +1,52 @@
+const chai = require('chai')
+const expect = chai.expect
+
+const { testFieldTypes } = require('../helper.js')
+
+const {
+  testHandlesNullData,
+  assertNumberOfRecords
+} = require('../integration_helper.js')
+
+const {
+  slug,
+  from,
+  to,
+  days,
+  numberOfDays,
+  formatDate
+} = require('../setup.js')
+
+describe('age_destroyed metric', async () => {
```

Comment:
Use the real function name - `SAN_AGE_DESTROYED`, also in other integration tests. Describe what you test.